extensions/adobe/experience/adcloud/advertisementdetails.example.1.json

```diff
@@ -1,9 +1,9 @@
 {
-  "https://ns.adobe.com/experience/adcloud/advertisementdetails/adId": 4116904851,
-  "https://ns.adobe.com/experience/adcloud/advertisementdetails/runtime": "HTML5",
-  "https://ns.adobe.com/experience/adcloud/advertisementdetails/class": "video",
-  "https://ns.adobe.com/experience/adcloud/advertisementdetails/adUnitType": "HTML5",
-  "https://ns.adobe.com/experience/adcloud/advertisementdetails/asset": {
+  "https://ns.adobe.com/experience/adcloud/adId": 4116904851,
+  "https://ns.adobe.com/experience/adcloud/runtime": "HTML5",
+  "https://ns.adobe.com/experience/adcloud/class": "video",
+  "https://ns.adobe.com/experience/adcloud/adUnitType": "HTML5",
+  "https://ns.adobe.com/experience/adcloud/asset": {
```

Comment:
there is no asset in experience/adcloud - this should be to the common one, yes?  Or change the name to assetdetails, so that it's clear what it is referencing..

Reply:
Changed name to assetDetails